docs/customising/config.md

```diff
@@ -994,6 +995,22 @@ SHARED_DIRECTORIES:
   </dd>
 
   <dt>
+    <a name="widget_page"><code>WIDGET_PAGE</code></a>
```

Comment:
To me this config key looks like it would accept a URL path. `ENABLE_WIDGETS` might be a bit clearer.